encoding/json/encoding_test.go

```diff
@@ -3221,91 +3221,3 @@ func TestExportFunctionValue(t *testing.T) {
         `,
 	)
 }
-
-// valueWithCachedTypeID recursively caches type ID of value v's type.
-// This is needed because each type ID is lazily cached on
-// its first use in ID() to avoid performance penalty.
-func valueWithCachedTypeID(v cadence.Value) cadence.Value {
```

Comment:
I moved this and the below method to `values.go` and `types.go` files respectively, to be able to use in other tests, and to avoid circular dependency.